org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -1281,7 +1281,11 @@ class String {
 	 * 		"become".contains("CO")      ==> Answers false
 	 */
 	method contains(other) {
-		return self.indexOf(other) > 0
+		try {
+			return self.indexOf(other) >= 0
+		} catch e {
```

Comment:
Sorry... I forgot we still can't pass classes or classnames as parameters and resolve them in runtime. It's the same problem that we had with #666.